modules/ROOT/pages/dw-core-functions-orderby.adoc

```diff
@@ -151,9 +152,77 @@ output application/json
 ]
 ----
 
+=== Example
+
+This example orders a JSON array payload descending by the type String. The expression orders them by type first, then reverses the order based on the `[-1 to 0]`.
```

Comment:
"example orders a JSON array payload descending by the type String." -> "example orders the objects in a JSON array."   (removed "payload" since we we're using a DW variable as input)

"expression orders them by type first, then" -> "expression first orders them alphabetically by the value of the `Type` key, then"  or similar